projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_base.py

```diff
@@ -14,7 +15,7 @@
 
 shared_test_values = {
     "test_payload1": {"key1": "val1", "key2": "val2", "key3": "val3"},
```

Comment:
You should do this in another PR. But I would like to extend the test to cover all possible types . Currently only int, string and now decimal are covered.

Reply:
will create story for this